static/app/views/insights/uptime/views/overview.spec.tsx

```diff
@@ -26,22 +27,17 @@ describe('Uptime Overview', function () {
   const project = ProjectFixture();
   const team = TeamFixture();
 
-  jest.mocked(usePageFilters).mockReturnValue({
-    isReady: true,
-    desyncedFilters: new Set(),
-    pinnedFilters: new Set(),
-    shouldPersist: true,
-    selection: {
-      datetime: {
-        period: '10d',
-        start: null,
-        end: null,
-        utc: false,
-      },
-      environments: [],
-      projects: [],
+  const selection = {
```

Comment:
```suggestion
  const selection: Partial<PageFiltersState> = {
```
but preferably, just leave this as `mockReturnValue(PageFilterStateFixture({...}))` as it will ensure that locality of the type errors is better.